franka_gazebo/src/franka_hw_sim.cpp

```diff
@@ -331,7 +336,7 @@ void FrankaHWSim::writeSim(ros::Time /*time*/, ros::Duration /*period*/) {
 
 void FrankaHWSim::eStopActive(bool /* active */) {}
 
-bool FrankaHWSim::readParameters(const ros::NodeHandle& nh, const urdf::Model& urdf) {
+auto FrankaHWSim::readParameters(const ros::NodeHandle& nh, const urdf::Model& urdf) -> bool {
```

Comment:
Applied by clang-tidy. I can remove these changes if you want.

Reply:
the const in the for loop can stay, however the trailing return types, look just look ugly imo. We currently use clang-tidy 7 in the CI. There are lots of things that could be fixed with a newer version of clang-tidy, however there are other lints like  modernize-use-trailing-return-type or readability-function-cognitive-complexity that you do not want to have. Therefore, we currently stick to clang-tidy 7.

Reply:
Strange that the extension does not pick up the right clang-tidy config. I reverted the clang-tidy changes manually.

Reply:
It is because we include modernize* and readability*, so if new checks appear they will be included by default. Thats why you need to choose the correct clang-tidy version, so just use clang-tidy 7.

Reply:
Ah over overlooked that in your previous answer. I'm on clang-tidy 10.